src/data/crops/corn.js

```diff
@@ -11,7 +13,7 @@ export const cornSeed = crop({
   cropType: cropType.CORN,
   cropTimetable: {
     [SEED]: 3,
-    [GROWING]: 7,
+    [GROWING]: [1, 1, 1, 2, 2],
```

Comment:
that's an interesting thought that had not occurred to me. so you're thinking to basically update this to `cropTimeline: [3, 1, 1, 1, 2, 2]` instead of `cropTimetable`, and then the "seed" images get renamed to "growing-1" (or maybe "growing-0" since i found out after i started daysWatered was 0 based when i assumed it was 1 based) since it now knows how to cycle through a series of images, and it doesn't really care what they're called 🤔 

is there anywhere in the code that relies on knowing something is in the seed phase outside of the images? i haven't looked yet but maybe you know offhand. if not this seems like a pretty straightforward change to make.

Reply:
well, are you thinking we would remove the concept of "SEED" stage entirely? or would we update `getLifeStageRange` to assume the 0 index of the cropTimeline is the "SEED" stage? 

i only see 1 place that isn't obvious to update if we go for completely remove SEED stage and that is in Plot.js where it sets the label text

Reply:
> well, are you thinking we would remove the concept of "SEED" stage entirely?

Yeah exactly. The `cropTimetable` is mostly for determining which image to show, and with this PR there isn't a meaningful or functional distinction between the initial planted "seed" image and the intermediate growth stage images.

Reply:
Oh whoops I just noticed your previous comment. 😅

> update this to `cropTimeline: [3, 1, 1, 1, 2, 2]` instead of `cropTimetable`

Yep!

> and then the "seed" images get renamed to "growing-1"

I don't think we need to rename the `*-seed` image files, as those are used in non-Field  contexts (like the Shop).

> is there anywhere in the code that relies on knowing something is in the seed phase outside of the images?

Nothing that comes to mind. Even if so, such logic could be refactored to work with `cropTimetable` being replaced with `cropTimeline`.

Reply:
done! everything seems to be working but please double check and test this 😄